Sources/SortedArray.swift

```diff
@@ -57,6 +57,14 @@ public struct SortedArray<Element> {
         _elements.append(contentsOf: newElements)
         _elements.sort(by: areInIncreasingOrder)
     }
+
+    /// Removes an element from the array, preserving the sort order.
+    ///
+    /// - Complexity: O(_log(n)_), where _n_ is the size of the array.
```

Comment:
The complexity of this is O(n), not O(log n). `Array.remove(at:)` is O(n), which will outweigh the O(log n) searching in the average case.